BookingSystem/Frontend/src/hotelSearch/actions/hotelSearchActions.js

```diff
@@ -25,6 +50,89 @@ class HotelSearchActions {
       })
       .catch(error => dispatch(fetchHotelsFailure(error)));
   };
+
+  static setCity = createAction(actionType.SET_CITY, city => ({
+    city: city
+  }));
+  static setCountry = createAction(actionType.SET_COUNTRY, country => ({
+    country: country
+  }));
+
+  static setCityOptions = createAction(actionType.SET_CITY_OPTIONS, data => ({
+    cityOptions: data
+  }));
+
+  static setCountryOptions = createAction(
+    actionType.SET_COUNTRY_OPTIONS,
+    data => ({
+      countryOptions: data
+    })
+  );
+
+  static loadCityOptions = search => dispatch => {
+    dispatch(loadCityOptionsRequest);
+    return HotelSearchService.fetchCities({
+      cityName: search.cityName,
+      countryName: search.countryName
+    })
```

Comment:
fix the indent please, it should be at the same level as .then for better readability